docs/protocol/common_problems.md

```diff
@@ -133,3 +133,7 @@ WARN sc_service::client::client: Block import error: Potential long-range attack
 ```
 The node somehow ended up being on a fork, try wiping and starting from scratch.
 
+### Still Facing Trouble? Take a look at our forums below
+
+<iframe src="https://forum.subspace.network/search?expanded=true&q=tags%3Afaq%20order%3Alatest" width="100%" height="600px" frameborder="0" scrolling="auto"></iframe>
```

Comment:
This does not work for me I'm afraid.

![image](https://github.com/subspace/subspace-docs/assets/11335994/f4966820-92a7-4a99-9351-db7d63e2705d)

Reply:
@ImmaZoni this now works for me with latest commit. We still need the functionality copied to latest.